app/views/facility_journals/show.html.haml

```diff
@@ -16,62 +16,58 @@
       = f.input :status_string
   .span4
     .well
-      - if Settings.financial.journal_format.xls
-        %p=link_to 'Download XLS File', @journal.file.url
-      - if Settings.financial.journal_format.xml
-        %p=link_to 'Download XML File', facility_journal_path(current_facility, @journal, :format => 'xml')
-      - if Settings.financial.journal_format.csv
-        %p=link_to 'Download CSV File', facility_journal_path(current_facility, @journal, :format => 'csv')
+      - %i(xls xml csv).each do |format|
+        - if Settings.financial.journal_format.public_send(format)
+          %p= link_to t(".download.#{format}"), facility_journal_path(current_facility, @journal, format: format)
 
 %hr
 
-%h3= t('.head.h3')
-%p= t('.instruct.orders')
+%h3= t(".head.h3")
+%p= t(".instruct.orders")
 
 - if @journal.open?
-  %p= t('.journal.pending')
+  %p= t(".journal.pending")
 - elsif !@journal.is_successful?
-  %p= t('.journal.failed')
-- elsif @journal.is_successful? && @journal.is_reconciled?
-  %p= t('.journal.success')
+  %p= t(".journal.failed")
+- elsif @journal.is_successful? && @journal.reconciled?
+  %p= t(".journal.success")
 - else
-  %p= t('.instruct.reconcile')
+  %p= t(".instruct.reconcile")
 
 - if @order_details.empty?
-  %p.notice= t('.notice')
+  %p.notice= t(".notice")
 - else
-  = form_tag facility_journal_reconcile_path(current_facility, @journal), :method => :post do
+  = form_tag facility_journal_reconcile_path(current_facility, @journal), method: :post do
     %table.table.table-striped.table-hover
       %thead
-        - is_submittable = @journal.is_successful? && !@journal.is_reconciled?
-        - if is_submittable
+        - if @journal.submittable?
```

Comment:
In the case of the `else` statement of `reconciled`, do we end up doing a lot of extra lookups because it runs the find every time this is called?